density/data/capacity_group.json

```diff
@@ -3,112 +3,156 @@
     {
       "capacity": 22, 
       "group_id": 147, 
-      "group_name": "Architectural and Fine Arts Library 1"
+      "group_name": "Architectural and Fine Arts Library 1",
+	  "parent_id": 146, 
```

Comment:
you're mixing tabs and spaces here.
it doesn't matter a ton but it'd be nice to keep this standard